src/components/projects/lean-workflow/ManuscriptPageContainerLW.tsx

```diff
@@ -222,16 +223,16 @@ const ManuscriptPageView: React.FC = () => {
                   which will happen on the very last step = 'Published'
                   this should be handled later with are more graceful way but it is not clear at this point how.
               */}
-                {submission?.nextStep && (
+                {/* {submission?.nextStep && (
                   <ManualFlowTransitioning
                     submission={submission}
                     userRole={getUserRole(project, user?.userID)}
                     documentId={`${project._id}#${manuscript._id}`}
                     hasPendingSuggestions={!!hasPendingSuggestions}
                   >
-                    <SaveStatusController isDirty={false} />
+                    <SaveStatusController />
                   </ManualFlowTransitioning>
-                )}
+                )} */}
```

Comment:
let's remove all that commented component as we are not going to use it in the editor

Reply:
removed